mavros/scripts/mavsys

```diff
@@ -78,6 +78,24 @@ def do_rate(args):
 
 
 
+def do_message_interval(args):
+
+    if args.id is None:
+        fault("id not specified")
+        return
```

Comment:
Perhaps better to add ability to specify mavlink message name in mavsys.
That can be done by using pymavlink.

Idea:
```python
try:
    from pymavlink.dialects.v20 import common as ml_common
except ImportError:
    ml_common = None


def find_id(name):
    rev = {v.name: k for k, v in ml_common.mavlink_map.items()}
    return rev[name.upper()]